### ISSUE TITLE
Permissions checkboxes have no label for screen reader users

### ISSUE THREAD

**Issue by thibaudcolas** [wagtail/wagtail#5329]
Related: #5328, #5274

In the permissions management of roles, Wagtail displays permission toggles as checkboxes in a table view:

<img width="629" alt="permissions-labels" src="https://user-images.githubusercontent.com/877585/58498051-25fcec00-8175-11e9-801c-63bfcec21d4a.png">

Those checkboxes don't have any label for screen reader users, making it hard / impossible to understand how to update permissions.

---

A good fix for this would be to provide a screen-reader-only label for each of those checkboxes, for example `can change user`, `can delete site`, etc.

**Comment by thibaudcolas** [comment]
Addressed in #5395.
